Documentation/app-container.md

```diff
@@ -33,8 +33,8 @@ To validate that `rkt` successfully implements the ACE part of the spec, use the
 # rkt --insecure-options=image run \
 	--mds-register \
 	--volume=database,kind=host,source=/tmp \
-	https://github.com/appc/spec/releases/download/v0.7.4/ace-validator-main.aci \
-	https://github.com/appc/spec/releases/download/v0.7.4/ace-validator-sidekick.aci
+	https://github.com/appc/spec/releases/download/v0.8.0/ace-validator-main.aci \
+	https://github.com/appc/spec/releases/download/v0.8.0/ace-validator-sidekick.aci
```

Comment:
It looks like those were simply missing in latest releases, @jonboulle just uploaded them for [0.8.6](https://github.com/appc/spec/releases/tag/v0.8.6), so it should be fine next time we bump appc.